### PR TITLE
Update/filterbar mobile view imporovments

### PR DESCRIPTION
**Pull request by enejb**
Updates to the mobile view. 
This PR tried to add a Portal for the mobile view that would display the checkboxes. 

image

image

### CODE REVIEW

**Comment by keoshi** [comment]
Looking good so far! Are you planning on modifying the behavior of the date selection too?

File: client/my-sites/activity/filterbar/date-range-selector.jsx [hunk 1]
```diff
@@ -212,15 +216,65 @@ export class DateRangeSelector extends Component {
...
+							} ) }
+						{ from &&
+							! to &&
+							translate( '{{icon/}} Please select the last day.', {
+								components: { icon: <Gridicon icon="info" /> },
+							} ) }
+						{ from &&
+							to && (
+								<Button borderless compact onClick={ this.handleResetSelection }>
+									{ translate( '{{icon/}} clear', {
```
Comment (a8ci18n) [inline client/my-sites/activity/filterbar/date-range-selector.jsx]: Hi! I've found a possible matching string that has already been translated **14** times:
`translate( '{{icon/}} clear dates' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by keoshi** [comment]
@enejb pushed the changes we talked about yesterday and a few more:

- Reviewed spacing around selections (type and dates).
- Changed the breaking point to be `>660px` instead of `>480px`.
- Made calendar tips font-size consistent.
- Added margin to reset filtering icon.
- Also tried to make the calendar take 100% of the available width, but unfortunately the CSS classes it offers don't allow us to solve the problem where ranges have spacings between them ([see this](https://user-images.githubusercontent.com/390760/45880013-7928ce80-bd9e-11e8-8094-9448a648d4a3.png)).

Think it's pretty solid right now, with one minor thing missing: the styles for the button's selected state are broken in desktop.

image

Could we add an additional class to the buttons that would allow us to target each state correctly? For example, in addition to `is-selected` would be good to have an `is-active` state, which would be used for when the selection is opened but there isn't a token yet. Does that make sense?

File: client/my-sites/activity/filterbar/date-range-selector.jsx [hunk 2]
```diff
@@ -83,16 +83,18 @@ export class DateRangeSelector extends Component {
...
 		if ( this.isSelectingFirstDay( fromDate, toDate, day ) ) {
+			alert( 'set first day' );
 			this.setState( {
 				fromDate: day,
 				enteredToDate: null,
 			} );
 			return;
 		}
-
+		alert( 'set second day' );
```
Comment (keoshi) [inline client/my-sites/activity/filterbar/date-range-selector.jsx]: Noting this should be removed before merging.

File: client/my-sites/activity/filterbar/date-range-selector.jsx [hunk 3]
```diff
@@ -83,16 +83,18 @@ export class DateRangeSelector extends Component {
 				enteredToDate: day,
 				toDate: day,
 			} );
+			alert( 'set second day?' );
 			return;
 		}
 		if ( this.isSelectingFirstDay( fromDate, toDate, day ) ) {
+			alert( 'set first day' );
```
Comment (keoshi) [inline client/my-sites/activity/filterbar/date-range-selector.jsx]: Noting this should be removed before merging.

File: client/my-sites/activity/filterbar/date-range-selector.jsx [hunk 4]
```diff
@@ -83,16 +83,18 @@ export class DateRangeSelector extends Component {
 				enteredToDate: day,
 				toDate: day,
 			} );
+			alert( 'set second day?' );
```
Comment (keoshi) [inline client/my-sites/activity/filterbar/date-range-selector.jsx]: Noting this should be removed before merging.

**Comment by roccotripaldi** [comment]
I think this needs a rebase to get all the latest changes.

**Pull request merged**